manual/source/Tutorial/DataStructures/Alignment/ScoringSchemes.rst

```diff
@@ -0,0 +1,187 @@
+To do so, we simply call the function :dox:`globalAlignment` and give as input parameters the ``align`` object and the Blosum62 scoring scheme.
+The globalAlignment function returns the score of the best alignment, which we store in the ``score`` variable that is then printed together with the corresponding alignment.
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp
+   :fragment: alignment
+
+The output of a global alignment implementing the Blosum62 scoring function is as follows:
+
+.. includefrags:: demos/tutorial/alignment/scoring_scheme_matrix.cpp.stdout
+
+.. note::
+   As can be noted the output of this scoring scheme is completely different with respect to the output generated with the simple scoring scheme confirming that the scoring scheme choice is one of the most important step to achieve high quality alignments.
+
+Gap Evaluation
+--------------
+
+In the previous sections we proposed two simple code examples useful to highlight the differences between two scoring schemes capable to evaluate match and mismatch events. In this section we will see the three gap models, implemented in the SeqAn library, to evaluate the insertion and deletion events.
+
+Linear Gap Model
+^^^^^^^^^^^^^^^^
+
+The easiest is the Linear gap model that considers, for the alignment score computation, the gap length (g) giving the possibility to evaluate with different scores gaps of different sizes;
+
+.. image:: linear.png
+   :width: 160px
+   :align: center
+
+This gap model is chosen as standard when only a gap value is provided in the scoring function or when the two provided gaps have the same value. For instance, this gap model as been adopted during the alignment computation of the two proposed examples.
+
+Affine Gap Model
+^^^^^^^^^^^^^^^^
+
+It has been proven that the first amino acid or nucleotide inserted/deleted (identified as gap open) found during the alignment operations is more significant, from a biological point of view, than the subsequent ones (called gap extension), making the so called Affine Gap model a viable solution for the alignment of biomolecules.
+Affine gap model that attribute different costs to the gap open (d) and the gap extension (e) events, is able to assign an higher penalty to the gap presence with respect to its relative length (g).
+
+.. image:: affine.png
+   :width: 240px
+   :align: center
+
+The Affine Gap model implemented in the DP alignment algorithms is however quite expensive both in terms of computational time as well as in terms of memory requirements with respect to other less demanding solutions such as the Linear Gap model application.
+
+Dynamic Gap Model
+^^^^^^^^^^^^^^^^^
+
+An optimised version of the Affine Gap model called Dynamic Gap Selector (DGS) designed by Urgese et al. :cite:`Urgese2014`. This new gap model can be used to reduce the computational time and the memory requirement while keeping the alignment scores close to those computed with the Affine Gap model.
```

Comment:
I think it's worthwhile to mention a little bit(at least) when there is a large discrepancy between Affine gap and Dynamic gap. Since the example code produces the exactly same result, I don't know why we do not set "DynamicGap" as a default mode despite computational benefits.

11  demos/tutorial/alignment/scoring_scheme_affine_dgs.cpp.stdout
@@ -0,0 +1,11 @@
 +ScoreAffine: -12
 +      0     .   
 +        TELKDD-
 +          ||   
 +        --LKTEL
 +
 +ScoreDynamic: -12
 +      0     .   
 +        TELKDD-
 +          ||   
 +        --LKTEL

I quickly have read below paper, and I'm thinking about this explanation from a user's perspective a little bit.
http://porto.polito.it/2527508/1/IWBBIO_2014_paper_143.pdf

".. The B quadrant contains those Maximum Alignment Scores that
probably correspond to couples of NTs or AAs located approximately at the end
of the two compared sequences. The increasing score differences detected are due
to the fact that at this positions, when both Qry and Sbj have high length sizes,
a conspicuous number of different alignment paths can be potentially explored.."

So if the length of the query/subject strings gets longer, the difference gets increased as well. 
Is this correct?